code/datums/uplink/highly visible and dangerous weapons.dm

```diff
@@ -112,4 +112,4 @@
 /datum/uplink_item/item/visible_weapons/custom_ka
 	name = "Kinetic Laser Assembly"
 	item_cost = 12
-	path = /obj/item/weapon/gun/custom_ka/frame01/illegal
\ No newline at end of file
+	path = /obj/item/weapon/gun/custom_ka/frame01/illegal
```

Comment:
Mind checking this file out from master, if you know how? Need to remove this diff, if possible.

Reply:
Should be fixed.